package.json

```diff
@@ -10,7 +10,10 @@
     "packages/*"
   ],
   "scripts": {
-    "watch": "yarn workspace foam-vscode watch"
+    "watch:vscode": "yarn workspace foam-vscode watch",
```

Comment:
I added these as it was easier to run scripts from the root directory. Happy to remove if this is not a good practice.

Reply:
looks good to me!